samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -187,3 +196,35 @@ HorizontalVirtualGridList.parameters = {
 export const WithButtonSpotlightGoesToCorrectTarget = () => <ButtonAndVirtualGridList />;
 
 WithButtonSpotlightGoesToCorrectTarget.storyName = 'with Button, Spotlight goes to correct target';
+
+export const HorizontalSquaredVirtualGridList = () => (
+	<VirtualGridList
+		dataSize={updateDataSize(number('dataSize', Config, defaultDataSize))}
+		direction="horizontal"
+		horizontalScrollbar="hidden"
+		itemRenderer={renderItemWithoutLabels}
+		itemSize={{
+			minWidth: ri.scale(number('minSize', Config, 804)),
+			minHeight: ri.scale(number('minSize', Config, 804))
+		}}
+		key={select('scrollMode', prop.scrollModeOption, Config)}
+		noScrollByWheel={boolean('noScrollByWheel', Config)}
+		onKeyDown={action('onKeyDown')}
+		onScrollStart={action('onScrollStart')}
+		onScrollStop={action('onScrollStop')}
+		scrollMode={select('scrollMode', prop.scrollModeOption, Config)}
+		spacing={ri.scale(number('spacing', Config, 0))}
+		style={{
+			width: ri.scaleToRem(804),
+			height: ri.scaleToRem(804),
+			backgroundColor: 'white'
+		}}
+		spotlightDisabled={boolean('spotlightDisabled', Config, false)}
```

Comment:
```suggestion
		spacing={ri.scale(number('spacing', Config, 0))}
		spotlightDisabled={boolean('spotlightDisabled', Config, false)}
		style={{
			width: ri.scaleToRem(804),
			height: ri.scaleToRem(804),
			backgroundColor: 'white'
		}}
```

Reply:
🆗 I've also fixed the above code since I copied and pasted it.